packages/design-system/package.json

```diff
@@ -63,9 +63,15 @@
     "@accelint/biome-config": "workspace:*",
     "@accelint/typescript-config": "workspace:*",
     "@accelint/vitest-config": "workspace:*",
+    "@chromatic-com/storybook": "^3.2.3",
```

Comment:
Are we actually integrating with this?

Reply:
not currently. maybe one day? will remove for now though

Reply:
removed: https://github.com/gohypergiant/standard-toolkit/pull/167/commits/5a076e8a950d0f4a9e7de269a5a37b6fd50be3e7